py/vtdb/grpc_vtgate_client.py

```diff
@@ -118,7 +116,8 @@ def rollback(self):
     finally:
       self.session = None
 
-  @vtgate_utils.exponential_backoff_retry((dbexceptions.TransientError))
+  @vtgate_utils.exponential_backoff_retry((dbexceptions.ThrottledError,
```

Comment:
Exponential backoff on a ThrottledError sounds dangerous. I assume getting throttled and retrying is very cheap.